data/report_templates/project_summary.md

```diff
@@ -71,48 +76,63 @@ This means that our services are subject to highly stringent quality control pro
 so that you can be sure that your data is of excellent quality.
 
 Library preparation
-:   [cross] Not Swedac Accredited
+:   {{ project.accredit.library_preparation }}
 
 Sequencing data
-:   [tick] Swedac Accredited
+:   {{ project.accredit.sequencing }}
 
-Data flow
-:   [tick] Swedac Accredited
+Data Processing
+:   {{ project.accredit.data_processing }}
 
-Data processing
-:   [tick] Swedac Accredited
+Data Analysis
+:   {{ project.accredit.data_analysis }}
 
```

Comment:
Yup, I thought we can do that in the code and just have something like

```{% if samples.display ‰}
....
{% endif %}
```

we can determine if display to be `Ture` or `False` based upon the threshold we decide. Cause sometime 6 samples might fit in perfectly and sometime even 4 samples might be too much. I think we should have some flexibility to choose. But yes, in the end both does the work :wink: